imports/ui/components/study_plan/SemModulesCardContainer.js

```diff
@@ -0,0 +1,16 @@
+import { Meteor } from 'meteor/meteor';
+import { createContainer } from 'meteor/react-meteor-data';
+import { getAllModulesInSemester } from '../../../api/crud-controller/module/methods';
+import SemModulesCard from './SemModulesCard';
+
+export default SemModulesCardContainer = createContainer((props) => {
```

Comment:
Good way to separate data from UI - using reactive data containers.
@walrys and I might not need to use Redux if this is sufficient enough for us to handle the DB entries we have